apps/studio/schemas/organizer.ts

```diff
@@ -0,0 +1,34 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "organizer",
+  icon: HeartIcon,
+  title: "Organizers",
+  type: "document",
+  fields: [
+    defineField({
+      name: "name",
+      title: "Name",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "pronouns",
+      title: "Pronouns",
+      type: "string",
+    }),
+    defineField({
+      name: "linkedinURL",
```

Comment:
Kinda a small detail, but if we do change this to a `person` schema, maybe revert back to the socials array? We can opt to only query and show LinkedIn URLs for the board page, while potentially displaying other socials elsewhere. Honestly I could go either way on this, but I do like the expendability.

Reply:
Since this is a professional site, I think it would be better to stick to only LinkedIn URLs. I guess the only exception in which we would display other social media links would be for the official Hack organization.

Reply:
I think I might just still be a bit biased on this. But, if we only plan on displaying LinkedIn URLs for all `person`s that makes sense.
Some examples I can think of are participant testimonials, and sponsor representatives. Would we want to link to other platforms in these cases?

Reply:
This would probably be a better question for marketing. From what I've seen, participant testimonials don't really include links to their personal socials, just their name and maybe their occupation. For sponsor representatives, they usually include their LinkedIn and maybe their work emails.

Reply:
Just stated a quick thread in the Slack marketing channel. They mentioned a required email field, and an optional website field for organizers.

I think it's probably okay to just have a email, website, and LinkedIn field, all optional?

Reply:
Resolved via Slack. LinkedIn, email, and website fields, all of which are optional, have been added in [`e710f77`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/e710f77bda6b55fcc26180dad0828658ad24bf4d)